test/sanity/validate-modules/ignore.txt

```diff
@@ -161,6 +161,7 @@ lib/ansible/modules/cloud/google/gcpubsub_facts.py E322
 lib/ansible/modules/cloud/google/gcpubsub_facts.py E324
 lib/ansible/modules/cloud/google/gcpubsub_facts.py E326
 lib/ansible/modules/cloud/google/gcspanner.py E322
+lib/ansible/modules/cloud/kubevirt/kubevirt_cdi_upload.py E203
```

Comment:
Why we can't fix this issue? AFAIK shouldn't be complex to fix,no?

Reply:
I'd need to take this and rewrite it: #32049, then get it merged, then modify this patch to use it. The first time this issue came came up (when merging `k8s_auth`) I figured that'd be my plan B, as that's quite a time investment.

Reply:
Oh, and `openshift` which we depend on uses `requests` (or was it `urllib3`?) anyways under the hood, so it's not like there's anything gained from the user's POV.